apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -199,7 +199,15 @@ k8s-pvc-shared:
 k8s-delete-pvc-shared:
 	kubectl delete -n ${K8S_NAMESPACE} --ignore-not-found pvc ${SHARED_PVC_NAME}
 
-k8s-shared-storage: k8s-pv-shared k8s-pvc-shared
+
+k8s-efs-setup-job:
+	j2 mdn-efs-setup-job.yaml.j2 | kubectl apply -n ${K8S_NAMESPACE} -f -
+
+k8s-delete-efs-setup-job:
+	kubectl delete -n ${K8S_NAMESPACE} --ignore-not-found job mdn-efs-setup
+
+# this runs and then deletes the efs-setup-job
+k8s-shared-storage: k8s-pv-shared k8s-pvc-shared k8s-efs-setup-job k8s-delete-efs-setup-job
```

Comment:
Jobs will retry automatically by default, indefinitely up to an [optional deadline](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-termination-and-cleanup), but we'll lose that behavior if you delete the job immediately without validating that it succeeded.